ringpop.go

```diff
@@ -226,12 +258,23 @@ type BootstrapOptions struct {
 
 // Bootstrap starts the Ringpop
 func (rp *Ringpop) Bootstrap(opts *BootstrapOptions) ([]string, error) {
+
```

Comment:
Yes, I contemplated this over the weekend. @dansimau I think you're in the large minority here and should remove the whitespaces.